### PR TITLE
Update user guide for increment, add, edit, delete, undo commands

### LINKED ISSUE #129: Update UG - Bulk Increment Salary, Add, Edit, Delete, Undo

### PR DESCRIPTION
**Pull request by Goh-Li-Ting**
closes #129

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -76,10 +76,7 @@ If you are an **experienced user**:
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+  e.g `INDEX [n/NAME] [p/PHONE]` can be used as `1 n/John Doe p/97328601` or as `1 n/John Doe`.
```
Comment (Weiennn) [inline docs/UserGuide.md:79]: Should we contextualize it with the command find?
so 
e.g `find [i/ID] [n/NAME] [p/PHONE] [e/EMAIL] [d/DEPARTMENT] [r/ROLE] [s/SALARY_LOWER_BOUND - SALARY_UPPER_BOUND] ` can be used as `find n/John Doe` or as `find d/finance`.

**Review by Weiennn — APPROVED** [review]
Other than the comment LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -101,61 +98,33 @@ Format: `help`
...
-* Name, Position, and Department up to 100 characters each
+* `add i/A00001 n/John Doe p/97328601 e/johndoe@example.com d/Marketing r/Manager s/5000` Adds an employee `John Doe` into the list.
+* `add i/A00002 n/Betsy Crowe p/82517642 e/betsycrowe@example.com d/Finance r/Intern s/1200` Adds an employee `Betsy Crowe` into the list.
 
-Expected output when employee added successfully:
-* `Employee EMPLOYEE_ID NAME successfully added!`
+### Listing all employees : `list`
 
-Expected output when add command fails:
-* Invalid Salary : `Please input a non-negative salary as a numerical value (e.g. 4000)`
```
Comment (ricketytoc) [inline docs/UserGuide.md:124]: Should we keep the expected output too?

**Review by ricketytoc — APPROVED** [review]
Other than that I think looks ok to merge!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -101,61 +98,33 @@ Format: `help`
...
-application will automatically assign a unique ID to the employee.
+Adds an employee to EmployeeManager.
 
-Format: `add [id/EMPLOYEE_ID] name/NAME position/POSITION department/DEPARTMENT salary/SALARY`
+Format: `add i/ID n/NAME p/PHONE e/EMAIL d/DEPARTMENT r/ROLE s/SALARY`
 
 Examples:
-* `add id/A035B name/Richard Hew Jia Liang position/Supervisor department/Marketing salary/5000`
-* `add name/Richard Hew Jia Liang position/Supervisor department/Marketing salary/5000`
-
```
[the remark this answers is not in the document]
Comment (Goh-Li-Ting) [reply docs/UserGuide.md:114]: We are shifting the constraints to the prefix summary section for easier reference as the constraints apply to other commands like `edit` too!

File: docs/UserGuide.md (hunk 2 above)
Comment (Goh-Li-Ting) [reply docs/UserGuide.md:124]: I feel like the expected output is not needed as it doesn't really help the user to use the application properly (plus past year user guides did not include the expected outputs). The constraints probably also give similar information as the expected output when the command fails.

**Pull request merged**